stable/kubernetes-dashboard/values.yaml

```diff
@@ -80,8 +80,14 @@ ingress:
   ##
   # annotations:
   #   kubernetes.io/ingress.class: nginx
-  #   nginx.ingress.kubernetes.io/secure-backends: "true"
+  ## Annotations below are relevant if you plan to use TLS with
+  ## enableInsecureLogin set to true
```

Comment:
Can you put those two lines under the acme annotation and state something like "if you plan to use TLS backend with enableInsecureLogin set to false (default)" ?

Reply:
The acme annotation is relevant to the use of TLS... but whatever ok I'll do that.

Reply:
AFAIK the acme annotation is useful in case of Ingress using TLS to autogenerate certificate through cert-manager, whereas our concern is about telling the ingress to use HTTPS to connect to the backend.

Reply:
Done

Reply:
My goal was more to make it clear that every annotations below that comment were relevant for TLS, and acme is relevant when you use TLS only.
I find it confusing to say:
```
# instruction relevant to TLS
## Comment that says "if you plan to use tls use the below"
# another instruction relevant to TLS
```